.gitignore

```diff
@@ -56,3 +56,6 @@ storybook-static
 
 # Non-Yarn lockfiles
 package-lock.json
+
+# SARIF outputer default directory
+sarif-test-results
```

Comment:
Should we ignore all `.sarif` files? That way they'll never be commited, no matter where they're output to?